fiat-ldap/src/main/java/com/netflix/spinnaker/fiat/roles/ldap/LdapUserRolesProvider.java

```diff
@@ -166,7 +166,7 @@ private String getUserFullDn(String userId) {
                 configProps.getUserSearchBase(), configProps.getUserSearchFilter(), formatArgs);
         partialUserDn = res.getDn().toString();
       } catch (IncorrectResultSizeDataAccessException e) {
-        log.error("Unable to find a single user entry", e);
+        log.error("Unable to find a single user entry for " + userId, e);
```

Comment:
Since this is using [slf4j](http://www.slf4j.org/faq.html#logging_performance) this should be

```java
log.error("Unable to find a single user entry for {}", e);
```

Reply:
error method has overloaded function definition as bellow. I am using that in this case.
```
  Log an exception (throwable) at the ERROR level with an accompanying message.
  Params:
    msg – the message accompanying the exception
    t – the exception (throwable) to log
public void error(String msg, Throwable t);
```